### PR TITLE
Fix Issue 598

### PR DESCRIPTION
**Pull request by JessyCatterwaul**
https://github.com/razeware/emitron-iOS/issues/598

Note: This is labeled with "iOS 15" not because it requires iOS 15, but because it conditionally does, via an `if #available(iOS 15.0, *)` block.

### CODE REVIEW

**Review by 0xTim — DISMISSED** [review]
LGTM

**Comment by 0xTim** [comment]
I suspect we'll need to wait until Xcode 13 is available on GH actions until we can merge this

**Comment by JessyCatterwaul** [comment]
See https://github.com/razeware/emitron-iOS/pull/611.

**Pull request closed without merging**
